.github/workflows/node.js.yml

```diff
@@ -36,4 +36,16 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-      - run: yarn lint
+      - uses: actions/checkout@v4
+        with:
+          repository: xyj-3/sarif-js-sdk
+          path: sarif-js-sdk
```

Comment:
If you lock the specific revision to checkout with `ref: e23ca2d8dc6d70a7959a2098eba8ac332de907bb`, I'm willing to accept this as-is and worry about how the dependency package is organized later. This will guarantee that only xyj-3/sarif-js-sdk@e23ca2d8dc6d70a7959a2098eba8ac332de907bb is installed, kind of like `package-lock.json` or `yarn.lock`.

Reply:
Updated